Documentation/Signoffs/Sign.md

```diff
@@ -0,0 +1,90 @@
+
+## Constraints
+
+1. Seven-Segment Display & Controls
+  -At least 1 foot in height and 6 inches in width (LED display)
+  -Microprocessor to control each LED and receive data
+
+2. Wireless Data Transfer Module to receive information
+
+3. Outdoor Physical Sign/Stand
+  -Text above Seven-Segment Display Stating “Open Spots Available”
+
+
+## Buildable Schematic
+
+![Figure 1. Physical Sign Schematic](../3D&#32;Models/3DSign.PNG)
+<div align="center"> Figure 2. Physical Sign PDF Schematic 
+<br />
+<div align="left">
+
+## Analysis
+	
+*Seven-Segment Display and Controls
+
+The Seven-Segment (SS) display needs to be large enough to be seen from the entrance into the parking lot. After lots of research, not many options are available for large seven segment displays, so the idea of creating a custom made large waterproof seven segment display came to light. This will consist of seven red 12 V LED light strips [2] per display, so 14 of them total. The chosen LED strips can be cut to the desired length for whatever application, which will roughly be 6 inches per strip. This will make each segment roughly 7 inches (W) x 13 inches (H). These will be configured and controlled much like typical SS displays minus the decimal point due to that not being necessary for this application. Each LED will then be controlled either being on or off by an Arduino Mega 2560 REV3 [A000067] [1]. This specific Arduino board has 54 digital I/O ports which is necessary for this application. Previously, an Arduino Uno board was chosen for this, but the wireless module must use the serial pins (TXD and RXD) for communication on the board. These pins share the same ports as two of the digital I/O ports. This would leave only 12 I/O pins available, while 14 are needed. Each output only sends a 5 V signal with 40 mA of current. This is not enough voltage or current for the LEDs, therefore a power transistor circuit will be used for each LED consisting of a IRLZ44N [6] power transistor, a 12V DC source, a 10 KOhm resistor, and necessary grounding. Each LED strip will then be connected to each Arduino Uno digital I/O pin. The necessary programming will then be in place to control each pin on the Arduino board. Please note that the 12 V source will be separate, and will be designed to meet the load requirements for the board, module, and each LED strip.
+
+![Figure 2. Arduino Circuit Diagram](../Electrical/Schematics/Sources/Display_Schematic.PNG)]
```

Comment:
This isn't buildable as there are a number of components that don't have part numbers or labels. Also, it is best if the different designs are all done in the same software. (Ideally) 

But it can be ok in this case to use a different software as this won't have electrical connections to the other systems.